demos/kotlin-app/src/main/kotlin/com/pusher/chatkitdemo/MainActivity.kt

```diff
@@ -46,7 +46,7 @@ class MainActivity : AppCompatActivity() {
         state = State.Idle
 
         launch {
-            state = app.rooms()
+            state = app.roomService()
```

Comment:
Is there a way of synchronously getting a list of the rooms that the current user is a member of, akin to: https://github.com/pusher/chatkit-client-js/blob/4da9cd6871ced4b9b6d830082a89d5fbfabdf21f/src/current-user.js#L82-L84

Associated question: does the SDK keep a store of the rooms that the current user is a member of?

Reply:
`RoomService` has a method called `fetchUserRooms`.
However, these `services` will slowly hide away as part of moving to using a state machine. 
So there will be a homologous `UserStateMachine` that will provide these as part of the state of the user.

Reply:
Sure, but will there be a synchronous way of getting the list of rooms or will that have to be done through a call to `fetchUserRooms`?